src/test/java/de/tum/cit/aet/artemis/core/user/util/UserUtilService.java

```diff
@@ -539,4 +545,13 @@ public void adjustUserGroupsToCustomGroups(String userPrefix, String userSuffix,
             userTestRepository.save(user);
         }
     }
+
+    public void createLearnerProfilesForUsers(String userPrefix) {
```

Comment:
Please move this into a new UtilService. The learner profile will have more functionality in the future. Furthermore, it's a subsystem that is not directly tied to the core functionality of the domain of a user.